### PR TITLE
Bump versions

### PR DESCRIPTION
**Pull request by chukka**
Signed-off-by: Ram <1331672+chukka@users.noreply.github.com>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by chukka** [comment]
@davidkarlsen @unguiculus Can this be merged ?

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.12
+FROM alpine:3.12.1
```
[the remark this answers is not in the document]
Comment (chukka) [reply Dockerfile:1]: @davidkarlsen  I think Image should be deterministic , don't want changing image every time , WDYT?

cc @rimusz @unguiculus

[the remark this answers is not in the document]
Comment (unguiculus) [reply Dockerfile:1]: @davidkarlsen I agree. I'd stick with `3.12` in order to always get the latest patches.

**Comment by davidkarlsen** [comment]
@chukka can you rebase off master please?

**Comment by chukka** [comment]
closing #299  and opening #304

**Pull request closed without merging**